pandas/util/_decorators.py

```diff
@@ -107,6 +108,16 @@ def _deprecate_kwarg(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
             old_arg_value = kwargs.pop(old_arg_name, None)
+
+            if new_arg_name is None and old_arg_value is not None:
+                msg = (
+                    "the '{old_name}' keyword is deprecated and will be removed in a future version"
```

Comment:
you might need to break this line (too long) for the linter

Reply:
Thanks for catching that.  I need to remember to run ``git diff master -u -- "*.py" | flake8 --diff``